examples/Receiver_Response.py

```diff
@@ -0,0 +1,164 @@
+""" Calculate and Fit Receiver Response to Sinc
+===============================================
+
+Two files are required for the following example:
+
+    File1 contains the analytic signal acquired on the GDS oscilloscope
+    directly output from the AFG output. Each node pertains to signal with a
+    different frequency (in kHz) which are fit to a complex function to extract
+    the power (in :math:`V^{2}`).
+    File2 contains the quadrature signal acquired on the receiver when the same
+    signal of File1 is injected into it. Each node pertains to signal with a
+    different frequency (in kHz) which are converted to a PSD. The power (in
+    :math:`dg^{2}`) is calculated from the peak of the convolved PSD.
+
+    The receiver response is then the ratio of :math:`dg^{2}(\\nu)` to
+    :math:`V^{2}(\\nu)` which is fit to a sinc function. The final plot shows
+    the square of the data with the fit.
+"""
+import numpy as np
+import pyspecdata as psd
+from sympy import symbols
+import sympy as sp
+import re
+
+
+def for_plot(thisdata):
+    """Converts :math:`dg^{2}/V^{2}` to :math:`dg/\\mu V`
+    and sets labels accordingly for plotting"""
+    # Take sqrt for units of $dg/V$
+    thisdata.data = np.sqrt(thisdata.data)
+    # Units of $dg/\mu V$
+    thisdata /= 1e6
+    thisdata.rename("nu_test", r"$\Delta \nu$")
```

Comment:
this should not happen here -- it should happen elsewhere when you subtract out the carrrier.

Btw, I usually do something like `Dnu_str = r'$\Delta \nu$'` to make my life easier (then just use `Dnu_str` (variable) as the dim name)